src/puppetlabs/pcp/protocol.clj

```diff
@@ -70,11 +70,15 @@
   {:id MessageId
    :targets [Uri]})
 
-(def ErrorMessage
+(def v1-ErrorMessage
```

Comment:
I think we need to keep this as `ErrorMessage` for now, for backwards compatibility.